src/main/java/com/exadel/service/impl/TrainingFeedbackServiceImpl.java

```diff
@@ -30,6 +30,8 @@ public TrainingFeedbackServiceImpl(TrainingFeedbackRepository trainingFeedbackRe
 
     @Override
     public Optional<TrainingFeedback> addTrainingFeedback(TrainingFeedback trainingFeedback) {
-        return Optional.ofNullable(trainingFeedbackRepository.save(trainingFeedback));
+        System.out.println(trainingFeedback);
```

Comment:
It's normal for that day. Panic try to stupid debug. Perhaps, it doesn't exist in the code now.
Today I've tried to push first exception handlings, but ended up with a number of merge conflicts with old code, so I postponed this until approving recent pull-requests. We are missing you!